content/en/docs/setup/additional-setup/gateway/index.md

```diff
@@ -0,0 +1,307 @@
+
+#### Dedicated application gateway
+
+In this model, an application namespace has its own dedicated gateway installation.
+This allows giving full control and ownership to a single namespace.
+This level of isolation can be helpful for critical applications that have strict performance or security requirements.
+
+{{< image width="50%" link="user-gateway.svg" caption="Dedicated application gateway" >}}
+
+Unless there is another load balancer in front of Istio, this typically means that each application will have its own IP address,
+which may complicate DNS configurations.
+
+## Upgrading gateways
+
+### In place upgrade
+
+Because gateways utilize pod injection, new gateway pods that are created will automatically be injected with the latest configuration, which includes the version.
+
+To pick up changes to the gateway configuration, the pods can simply be restarted, using commands such as `kubectl rollout restart deployment`.
+
+If you would like to change the [control plane revision](/docs/setup/upgrade/canary/) in use by the gateway, you can set the `istio.io/rev` label on the gateway Deployment, which will also trigger a rolling restart.
+
+{{< image width="50%" link="inplace-upgrade.svg" caption="In place upgrade in progress" >}}
+
+### Canary upgrade (advanced)
+
+This upgrade method depends on using [control plane revisions](/docs/setup/upgrade/canary/).
+
+If you would like to more slowly control a rollout of a new revision, we can run multiple versions of the gateway deployment.
+For example, if I want to do a controlled rollout of the `canary` revision, create a copy of your current deployment with the `istio.io/rev=canary` label set:
+
+{{< text yaml >}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: istio-ingressgateway-canary
+  namespace: istio-ingress
+spec:
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  template:
+    metadata:
+      annotations:
+        inject.istio.io/templates: gateway
+      labels:
+        istio: ingressgateway
+        istio.io/rev: canary # Set to the control plane revision you want to deploy
+    spec:
+      containers:
+      - name: istio-proxy
+        image: auto
+{{< /text >}}
+
+When this is created, we will now have two versions of the gateway, both selected by the same Service:
+
+{{< text bash >}}
+$ kubectl get endpoints -o "custom-columns=NAME:.metadata.name,PODS:.subsets[*].addresses[*].targetRef.name"
+NAME                   PODS
+istio-ingressgateway   istio-ingressgateway-788854c955-8gv96,istio-ingressgateway-canary-b78944cbd-mq2qf
+{{< /text >}}
+
+{{< image width="50%" link="canary-upgrade.svg" caption="Canary upgrade in progress" >}}
+
+Unlike when deploying your own applications, you cannot utilize [Istio traffic shifting](/docs/tasks/traffic-management/traffic-shifting/) to control, as the traffic to the gateway is coming from external clients that we do not control.
```

Comment:
```suggestion
Unlike application services deployed inside the mesh, you cannot use [Istio traffic shifting](/docs/tasks/traffic-management/traffic-shifting/) to distribute the traffic between the gateway versions because their traffic is coming directly from external clients that Istio does not control.
```